data/na/swagger/1-0-5.yaml

```diff
@@ -1985,11 +2003,14 @@ definitions:
   CardResponse:
     description: 'For a 3D Secure payment process.'
     required:
-      - pa_res
+      - cres
```

Comment:
pa_res will be required for 3DSv1 still.
For 3DSv2 I think we'll also need the 'threeDSSessionData' value.